plugin-build/build.gradle.kts

```diff
@@ -10,13 +10,19 @@ plugins {
     id("java-gradle-plugin")
     id("com.vanniktech.maven.publish") version BuildPluginsVersion.MAVEN_PUBLISH apply false
     id("org.jlleitschuh.gradle.ktlint") version BuildPluginsVersion.KTLINT
+    // we need this plugin in order to include .aar dependencies into a pure java project, which the gradle plugin is
+    id("com.stepango.aar2jar") version BuildPluginsVersion.AAR_2_JAR
 }
 
 repositories {
     mavenLocal()
     mavenCentral()
     google()
 }
+apply(from = "gradle/bootstrap-sdk.gradle.kts")
```

Comment:
As a side note: I would not use `apply(from = ...)` if possible.

You can just place the function inside `buildSrc` and have it available to use.